mackerel-plugin-mysql/lib/mysql.go

```diff
@@ -46,14 +46,15 @@ func (m MySQLPlugin) defaultGraphdef() map[string]mp.Graphs {
 			Unit:  "float",
 			Metrics: []mp.Metrics{
 				{Name: "Com_insert", Label: "Insert", Diff: true, Stacked: true, Type: "uint64"},
+				{Name: "Com_insert_select", Label: "Insert Select", Diff: true, Stacked: true, Type: "uint64"},
 				{Name: "Com_select", Label: "Select", Diff: true, Stacked: true, Type: "uint64"},
 				{Name: "Com_update", Label: "Update", Diff: true, Stacked: true, Type: "uint64"},
 				{Name: "Com_update_multi", Label: "Update Multi", Diff: true, Stacked: true, Type: "uint64"},
 				{Name: "Com_delete", Label: "Delete", Diff: true, Stacked: true, Type: "uint64"},
 				{Name: "Com_delete_multi", Label: "Delete Multi", Diff: true, Stacked: true, Type: "uint64"},
 				{Name: "Com_replace", Label: "Replace", Diff: true, Stacked: true, Type: "uint64"},
-				{Name: "Com_set_option", Label: "Set Option", Diff: true, Stacked: true, Type: "uint64"},
-				{Name: "Qcache_hits", Label: "Query Cache Hits", Diff: true, Stacked: false, Type: "uint64"},
```

Comment:
I think it is copy&paste issue because Qcache_* is bundled to graph for query cache.
https://github.com/mackerelio/mackerel-agent-plugins/blob/2ba382fdfe71ec75b144434c8e2379a9ece9ae2f/mackerel-plugin-mysql/lib/mysql.go#L517

Reply:
Discussed with our SRE team.
Number of requested `SELECT` queries are sum of `Qcache_hits` and `Com_select` (As mentioned in https://github.com/mackerelio/mackerel-agent-plugins/pull/390#discussion_r135687312), so putting them into one graph is useful.  Can you revert it?
(And the Qcache_* graph is good to monitor overall status of query caching.)

Reply:
reverted in 528d243